boxes/generators/trafficlight.py

```diff
@@ -62,6 +62,9 @@ def __init__(self):
         self.argparser.add_argument(
             "--upright",  action="store", type=boolarg, default=True,
             help="stack lights upright (or side by side)")
+            
```

Comment:
I wonder if we can get away with only one argument and don't add holes if it is left empty or one of the diameter(s?) are zero. This is how we handle it with several other features already. Having the holes enabled by default is fine IMHO. For one it brings attention to this features to the users. It is also kinda necessary to have a good example on how to use it.

Given the format is more complex than just a number one could add a section in the docs and add an Info link as for some other parameters. But that's not strictly necessary.